### PR TITLE
Add tips to `--ffi` reference

### PR DESCRIPTION
**Pull request by odyslam**

### CODE REVIEW

File: src/cheatcodes/ffi.md [hunk 1]
```diff
@@ -12,6 +12,13 @@ Calls an arbitrary command if [`ffi`](../reference/config.md#ffi) is enabled.
 
 It is generally advised to use this cheat code as a last resort, and to not enable it by default, as anyone who can change the tests of a project will be able to execute arbitrary commands on devices that run the tests.
 
+### Tips
+
+- Make sure that the output is `abi-encoded`
```
Comment (onbjerg) [inline src/cheatcodes/ffi.md:17]:
```suggestion
- Make sure that the output is ABI encoded
```

File: src/cheatcodes/ffi.md [hunk 2]
```diff
@@ -12,6 +12,13 @@ Calls an arbitrary command if [`ffi`](../reference/config.md#ffi) is enabled.
...
+- Make sure that the output does not include a `\n` newline character. (e.g in rust use `print!` vs `println!`)
+- Remember that the script will be executed from the `top-level` directory of your project, not inside `test`
```
Comment (onbjerg) [inline src/cheatcodes/ffi.md:19]:
```suggestion
- Remember that the script will be executed from the top-level directory of your project, not inside `test`
```

**Review by onbjerg — APPROVED** [review]
minor nits

**Comment by odyslam** [comment]
@onbjerg I cant' merge

**Pull request merged**